crates/katana/core/src/service/messaging/starknet.rs

```diff
@@ -201,19 +199,16 @@ impl Messenger for StarknetMessaging {
             .map_err(|_| Error::SendError)
             .unwrap()
             .iter()
-            .for_each(|(block_number, block_events)| {
+            .for_each(|e| {
                 debug!(
                     target: LOG_TARGET,
-                    block_number = %block_number,
-                    events_count = %block_events.len(),
+                    event = ?e,
                     "Converting events of block into L1HandlerTx."
```

Comment:
Message could be updated here for something more accurate.
```suggestion
                    "Converting event into L1HandlerTx."
```